src/components/KeyboardAvoidingView/index.js

```diff
@@ -0,0 +1,14 @@
+/*
+ * The KeyboardAvoidingView is only used on ios
+ */
+import React from 'react';
+import {View} from 'react-native';
+
+const KeyboardAvoidingView = props => (
+    // eslint-disable-next-line react/jsx-props-no-spreading
+    <View {...props} />
```

Comment:
Should this be sure to omit any props that could be added only for the KAV? For example, I think `behavior` is a prop that we don't want to set on a `<View>`. There are perhaps others.

Reply:
Left a comment. We should omit the others too.